src/types/libdef/npm-custom/@testing-library/react_v11.x.x.js

```diff
@@ -1,11 +1,170 @@
+// This was copied from https://github.com/flow-typed/flow-typed/blob/2997a88721d3ed0057cdb87e363c5d01b7372998/definitions/npm/%40testing-library/react_v11.x.x/flow_v0.104.x-/react_v11.x.x.js
+// And adapted for our version of Flow
 // @flow
-/* eslint-disable flowtype/no-existential-type */
-
-// This file is copied from:
-// I couldn't figure out how to get the CLI to accept the proper arguments.
-// https://github.com/flow-typed/flow-typed/blob/8c09c1003c692c895f150bd6b3fa4b2d50569715/definitions/npm/%40testing-library/react_v10.x.x/flow_v0.104.x-/react_v10.x.x.js
 
+/**
+ * A local copy from:
+ * https://github.com/A11yance/aria-query/blob/2e6a3011a0d8987655f3a14853934fe3df38a8d8/flow/aria.js
+ */
 declare module '@testing-library/react' {
+  // JULIEN: I moved all the ARIA roles inside the module.
```

Comment:
The original libdef had all these roles in a separate "module", but that wasn't working for us (and I've a lot of doubt that it works on any version of Flow actually).